src/OrchardCore.Modules/Orchard.Themes/Orchard.Themes.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\OrchardCore\Microsoft.AspNetCore.Modules.Abstractions\Microsoft.AspNetCore.Modules.Abstractions.csproj" />
+    <ProjectReference Include="..\..\OrchardCore\OrchardCore.AsModule\OrchardCore.AsModule.csproj" />
```

Comment:
Shouldn't it reference AsTheme? It's doing the same right now because we have a manifest file, but it's about semantics.